test/JDBC/expected/fts-contains-multicol-vu-verify.out

```diff
@@ -0,0 +1,1339 @@
+895#!#The Industrial Revolution transformed manufacturing processes globally.#!#Photosynthesis converts light energy into chemical energy in plants.#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.#!#The Internet of Things connects everyday devices to the network.
+896#!#The Industrial Revolution transformed manufacturing processes globally.#!#Photosynthesis converts light energy into chemical energy in plants.#!#The Internet of Things connects everyday devices to the network.#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.
+897#!#The Industrial Revolution transformed manufacturing processes globally.#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.#!#Photosynthesis converts light energy into chemical energy in plants.#!#The Internet of Things connects everyday devices to the network.
+898#!#The Industrial Revolution transformed manufacturing processes globally.#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.#!#The Internet of Things connects everyday devices to the network.#!#Photosynthesis converts light energy into chemical energy in plants.
+899#!#The Industrial Revolution transformed manufacturing processes globally.#!#The Internet of Things connects everyday devices to the network.#!#Photosynthesis converts light energy into chemical energy in plants.#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.
+900#!#The Industrial Revolution transformed manufacturing processes globally.#!#The Internet of Things connects everyday devices to the network.#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.#!#Photosynthesis converts light energy into chemical energy in plants.
+901#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.#!#Photosynthesis converts light energy into chemical energy in plants.#!#The Industrial Revolution transformed manufacturing processes globally.#!#The Internet of Things connects everyday devices to the network.
+906#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.#!#The Internet of Things connects everyday devices to the network.#!#The Industrial Revolution transformed manufacturing processes globally.#!#Photosynthesis converts light energy into chemical energy in plants.
+907#!#The Internet of Things connects everyday devices to the network.#!#Photosynthesis converts light energy into chemical energy in plants.#!#The Industrial Revolution transformed manufacturing processes globally.#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.
+912#!#The Internet of Things connects everyday devices to the network.#!#Shakespeare wrote 37 plays and 154 sonnets during his lifetime.#!#The Industrial Revolution transformed manufacturing processes globally.#!#Photosynthesis converts light energy into chemical energy in plants.
+~~END~~
+
+
+-- negative test
+-- should throw an error as the columns txt3 and txt4 are not fulltext indexed, BUT DOES NOT
```

Comment:
Better to remove this since this is not an expected behavior and is a bug, for negative tests, we need test cases to handle cases like - non existent column or wrong column name, null values, empty search string,